disco_aws_automation/disco_deploy.py

```diff
@@ -723,7 +723,15 @@ def test_ami(self, ami, dry_run, deployment_strategy=None):
             desired_deployment_strategy = deployment_strategy
         else:
             desired_deployment_strategy = self.hostclass_option_default(hostclass, 'deployment_strategy',
-                                                                        DEPLOYMENT_STRATEGY_CLASSIC)
+                                                                        DEPLOYMENT_STRATEGY_BLUE_GREEN)
+
+        if desired_deployment_strategy == DEPLOYMENT_STRATEGY_CLASSIC:
+            logger.warning(
+                "Classic deployment will be removed in a future version of Asiaq."
+                "Please switch to Blue/Green deployment by setting `deployment_strategy=%s` under your "
+                "hostclass' section in disco_aws.ini",
+                desired_deployment_strategy
+            )
```

Comment:
Won't this tell people to use `deployment_strategy=classic` instead of `blue_green`?